File: Frontend/src/app/classes/acesso-aprovacoes/periodo-ausencia-usuario.ts
export class PeriodoAusenciaUsuario {
    codigo: number = 0;
    usuario: string = '';
    tipoLiberacao: string = '';
    tipoAusencia: string = '';
    inicioPeriodo: Date = new Date();
    fimPeriodo: Date = new Date();
    criadoEm: Date = new Date();
    atualizadoEm: Date = new Date();
    checked: boolean = false;
}

File: Backend/TopSys.TopConWeb.Domain/Entities/LiberacaoAcesso/PeriodoAusenciaUsuario.cs
using System;

namespace TopSys.TopConWeb.Domain.Entities.LiberacaoAcesso
{
    public class PeriodoAusenciaUsuario
    {
        public int Codigo { get; set; }

        public string Usuario { get; set; }

        public string TipoLiberacao { get; set; }

        public string TipoAusencia { get; set; }

        public DateTime InicioPeriodo { get; set; }

        public DateTime FimPeriodo { get; set; }

        public DateTime? CriadoEm { get; set; }

        public DateTime? AtualizadoEm { get; set; }
    }
}
